src/WaterPokemon.java

```diff
@@ -0,0 +1,127 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class WaterPokemon extends Pokemon {
+    private final List<String> attacks = Arrays.asList("surf", "hydroPump", "hydroCanon", "rainDance");
+
+    public WaterPokemon(String name, int level, int hp, String food, String sound) {
+        super(name, level, hp, food, sound);
+    }
+
+    @Override
+    public String getType() {
+        return "water";
```

Comment:
Dit werkt, maar het is handiger om toch een "type" variabele bovenaan je klasse te zetten. Als je dan later nog het type wilt wijzigen van "water" naar "liquid", dan weet je dat je bovenaan de klasse de variabele kunt wijzigen. 
(Als je dat voor elke klasse standaard zo doet, dan wordt dat een automatisme)

In plaats van een variabele, mag je het ook een constante noemen, omdat het natuurlijk niet veranderd. In dat geval is het conventie om de naam in hoofdletter te schrijven en er het "final" keyword voor te zetten. Je kunt het zelfs static maken, omdat het voor elke instantie van deze klasse hetzelfde is. Dan wordt het dus `private static final String TYPE = "water";`.